docs/usage/casting.md

```diff
@@ -1,57 +1,123 @@
+
+```yaml
+type: custom:frigate-card
+cameras:
+  - camera_entity: camera.office
+    live_provider: go2rtc
+    cast:
+      method: standard
+menu:
+  media_player:
+    enabled: true
+```
+
+It will send the camera stream to the Google Cast device through [Media Browser](https://www.home-assistant.io/integrations/media_source/#playing-media-from-a-media-source) (the `media_player.play_media` action).
 
-## The `casting` profile
+The main disavanatage of this method is that the stream will be played through HLS, which has a **delay of around 10 seconds**. Not ideal for live monitoring.
 
-The optional [casting profile](../configuration/profiles.md?id=casting) provides
-some defaults to improve your casting experience. Use it like:
+## Dashboard Casting
+
+This method requires further configuration, but allows a **real-time experience**. This utilizes [Home Assistant Cast](https://cast.home-assistant.io).
 
 ```yaml
-profiles:
-  - casting
+type: custom:frigate-card
+cameras:
+  - camera_entity: camera.office
+    live_provider: go2rtc
+    cast:
+      method: dashboard
+      dashboard:
+        dashboard_path: dashboard-cast
+        view_path: office
 ```
 
-## Recommended configuration for Nest Hub
+When the cast button is clicked, the card will perform the following Home Assistant action:
+
+```yaml
+action: cast.show_lovelace_view
+data:
+  entity_id: media_player.kitchen
+  dashboard_path: dashboard-cast
+  view_path: office
+```
+
+Which in turn will cast the specified Home Assistant dashboard (and view) to the Google Cast device.
+
+For the example above to work, a dashboard with `dashboard-cast` slug must exist:
+
+![image](https://github.com/user-attachments/assets/67f0e145-df05-412a-8c6d-897feb5439d2)
+
+And a view inside of it with `office` slug must exist. Inside the view, you can place the Frigate card configured for that camera for a proper full-screen casting experience:
+
+```yaml
+views:
+  - title: Office
+    type: panel
+    path: office
+    cards:
+      - type: custom:frigate-card
+        cameras:
+          - camera_entity: camera.office
+            live_provider: go2rtc
+        profiles:
+          - low-performance
+          - casting
+```
+
+The `casting` profile pre-configures the card to be casted to a 16:9 screen (such as a TV), including hiding all interactive elements and the menu.
+
+The `low-performance` profile is also recommended, as normally casting devices have limited hardware capabilities.
 
-Using a `panel` dashboard with the following base configuration will result in
-the card consuming the entire device screen:
+![](https://github.com/user-attachments/assets/bd96c4ad-36f5-4501-9018-23b496e7edc5)
 
-### Configuration
+When casting to a Google Nest Hub, the following configuration can be used:
 
 ```yaml
 type: custom:frigate-card
 cameras:
   - camera_entity: camera.office
     live_provider: go2rtc
+profiles:
+  - low-performance
+  - casting
+menu:
+  style: outside
```

Comment:
```suggestion
  style: hidden
```

Reply:
Are you sure? `menu.style: outside` was intentional, given I'm disabling the menu in the `casting` profile again.

Re-enabling the menu is useful for Nest Hub, given it's touch-enabled.

Reply:
Well, I was just putting it back to the default. Do you think `outside` is better than `hidden` on NestHub devices for some reason?

Reply:
Huh. Do you have your Frigate menu button turned off? `hidden` should just show that single button, and the menu "expands" when it's clicked.

Reply:
Ah ok, that explains it, lol.